op-geth.yaml

```diff
@@ -0,0 +1,129 @@
+package:
+  name: op-geth
+  version: "1.101411.6"
+  epoch: 0
+  description: The go-ethereum command line interface
+  copyright:
+    - license: LGPL-3.0-or-later
+  dependencies:
+    runtime:
+      - busybox
```

Comment:
Do we need busybox or would coreutils + other utilities work here?

Reply:
Yep make the require changes